File: ShoeStore/ShoeStore.Domain/Entities/Notification.cs
namespace ShoeStore.Domain.Entities;

public class Notification
{
    public long Id { get; set; }
    public string? Code { get; set; }
    public string Title { get; set; } = null!;
    public string Message { get; set; } = null!;
    public string? Type { get; set; }
    public DateTime CreatedAt { get; set; }
}

File: shoestorefe/lib/domain/entities/notification.dart
class Notification {
  final int id;
  final String? code;
  final String title;
  final String message;
  final String? type;
  final DateTime createdAt;

  Notification({
    required this.id,
    this.code,
    required this.title,
    required this.message,
    this.type,
    required this.createdAt,
  });
}
